compiler/bindings/python/iree/compiler/tools/import_onnx/__main__.py

```diff
@@ -132,6 +340,29 @@ def parse_arguments(argv=None) -> argparse.Namespace:
         " Defaults to the directory of the input file.",
         type=Path,
     )
+    parser.add_argument(
+        "--opset-version",
+        help="Allows specification of a newer opset_version to update the model"
+        " to before importing to MLIR. This can sometime assist with shape inference.",
+        type=int,
+    )
+    parser.add_argument(
+        "--max-numel",
+        help="Maximum number of elements allowed in an inlined parameter constant.",
+        type=int,
+        default=100,
+    )
```

Comment:
Yeah, I definitely think this should be renamed to `min-numel` with help description 

```python
     help="Minimum number of elements for an initializer to be externalized. Only has an effect if 'externalize-params' is true.",
```